.github/workflows/python-publish.yml

```diff
@@ -31,3 +29,14 @@ jobs:
       uses: pypa/gh-action-pypi-publish@master
       with:
         password: ${{ secrets.PYPI_PASSWORD }}
+
+  dispatch:
+    needs: deploy
+    runs-on: ubuntu-latest
+    steps:
+      - name: Repository Dispatch
+        uses: peter-evans/repository-dispatch@v1
+        with:
+          token: ${{ secrets.MLCOMMONS_REPO_ACCESS }}
```

Comment:
We seem to be using a lot of discrete secrets. Do we want to document that in the `.github/workflows/README.md`?

Reply:
will do as a new PR https://github.com/mlperf/mlbox/issues/94